_source/_posts/2020-12-07-spring-cloud-config.md

```diff
@@ -100,39 +100,12 @@ public class CloudConfigServerApplication { ... }
 
 ## Create an OpenID Connect Application
 
-Sign up for a free developer account at <https://developer.okta.com/signup>. This will be used to secure your microservices using OAuth 2.0 and OpenID Connect (OIDC). After signing up, log in to your Okta account at https://your-okta-domain.okta.com.
-
-Click **Applications** in the top nav menu.
-
-{% img blog/spring-cloud-config/applications.png alt:"Click the Applications button" width:"800" %}{: .center-image }
-
-Click **Add Application**.
-
-{% img blog/spring-cloud-config/add-application.png alt:"Click Add Application" width:"800" %}{: .center-image }
-
-Select **Web** and click **Next**.
-
-{% img blog/spring-cloud-config/select-web.png alt:"Select Web and click Next" width:"800" %}{: .center-image }
-
-In **Application Settings** fill in the following values:
-
-- **Name**: `My Spring Cloud App` (or another name of your choosing)
-- **Base URIs**: `http://localhost:8001` and `http://localhost:8002`
-- **Login Redirect URIs**: `http://localhost:8001/login/oauth2/code/okta` and `http://localhost:8002/login/oauth2/code/okta`
-- **Logout Redirect URIs**: `http://localhost:8001` and `http://localhost:8002`
-- **Group Assignments**: `Everyone` (should be selected by default)
-- **Grant type allowed**: `Authorization Code`
-
-{% img blog/spring-cloud-config/application-settings.png alt:"Application settings" width:"800" %}{: .center-image }
-
-Click **Done**.
-
-{% img blog/spring-cloud-config/done.png alt:"Click Done" width:"800" %}{: .center-image }
+{% include setup/cli.md type="web"
```

Comment:
@bdemers FYI... I couldn't use `framework="Okta Spring Boot Starter"` here. If I did, it'd write Okta app key/values to `application.properties` (in the config server's classpath). I don't think it'd affect anything because config server doesn't have the Okta starter in the classpath, but it's not exactly clean either.